src/v1_0/rws_progress.cpp

```diff
@@ -0,0 +1,14 @@
+#include <include/abb_librws/v1_0/rws_progress.h>
+
+namespace abb :: rws :: v1_0 :: progress
+{
+    ProgressInfo RWSClient::getProgress(std::string const& id)
+    {
+        std::string uri = "/progress/" + id;
+        RWSResult rws_result = parseContent(httpGet(uri,{Poco::Net::HTTPResponse::HTTP_OK, Poco::Net::HTTPResponse::HTTP_CREATED}));
+        ProgressInfo progress_info;
+        progress_info.code = std::stoi(xmlNodeTextByTagAndAttribute(rws_result, "span", abb::rws::XMLAttribute("class","code")));
+        progress_info.state = xmlNodeTextByTagAndAttribute(rws_result, "span", abb::rws::XMLAttribute("class","state"));
+        return progress_info;
+    }
```

Comment:
You have made `getProgress()` a free function, how does this compile? Did you forget to add `rws_progress.cpp` to `CMakeLists.txt`?